Sources/Citadel/SFTP/SFTPClient.swift

```diff
@@ -276,48 +276,3 @@ final class SFTPResponses {
         }
     }
 }
-
-final class SFTPInboundHandler: ChannelInboundHandler {
-    typealias InboundIn = SFTPMessage
-    
-    let responses: SFTPResponses
-    let logger: Logger
-    
-    init(responses: SFTPResponses, logger: Logger) {
-        self.responses = responses
-        self.logger = logger
-    }
-    
-    func channelRead(context: ChannelHandlerContext, data: NIOAny) {
-        let message = unwrapInboundIn(data)
-        
-        self.logger.trace("SFTP IN:  \(message.debugDescription)")
```

Comment:
Is this logging line no longer needed in the renamed SFTPClientInboundHandler? (It seems like a copy and rename and this is the only line that is missing in that new file)